azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
+  description      VARCHAR(2048),
+  active           BOOLEAN,
+  deployable       VARCHAR(64),
+  created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by       VARCHAR(64)     NOT NULL,
+  modified_on      TIMESTAMP       DEFAULT NULL,
+  modified_by      VARCHAR(64)
+);
+
+CREATE TABLE IF NOT EXISTS image_versions (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  path             VARCHAR(1024) NOT NULL,
```

Comment:
Can you please verify if path is a keyword used in mysql? Is path allowed to be a column name?

Reply:
Yes path is allowed.